src/components/MultiLink/Legend.vue

```diff
@@ -226,7 +226,12 @@ export default Vue.extend({
       });
     },
 
+    // TODO: #176 use table name for var selection
```

Comment:
The full link to GitHub would be better than just the `#176`.

Reply:
See a081dfa